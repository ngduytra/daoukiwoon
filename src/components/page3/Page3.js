import React from 'react';
import './page3.styles.scss'
import Computer2 from '../../assets/img/Computer2.png'

const Page3 = () => {
    return (
        <div id='index-3'>
            <div className="content">
                <div className='left'>
                    <div className='title'>ACCOUNTING</div>
                    <div className='subtitle'>Accounting</div>
                    <div className='button'>G-Book</div>
                </div>
                <img src={Computer2} />
            </div>
        </div>
    )
}

export default Page3;