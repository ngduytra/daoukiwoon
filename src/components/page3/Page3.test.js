import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Page3 from './Page3';

configure({adapter: new Adapter()});

it('expect to render Page3 component', ()=>{
    expect(shallow(<Page3/>)).toMatchSnapshot();
})