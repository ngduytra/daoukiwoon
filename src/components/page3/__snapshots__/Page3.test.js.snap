// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Page3 component 1`] = `ShallowWrapper {}`;
